docs/api/source/index.rst

```diff
@@ -13,7 +13,24 @@ For Solid Servers that support notifications, they are sent when a resource is
 created, updated, deleted, or when a child resource is added to or removed from
 a container.
 
-**This library currently only supports notifications sent via WebSockets.**
+.. admonition:: Supported Protocols
```

Comment:
This change looks like this: 

<img width="808" alt="Screenshot 2022-04-20 at 01 17 03" src="https://user-images.githubusercontent.com/30827/164116449-364b3318-15bd-4d3c-a254-6fd82513c4b3.png">